src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -100,7 +92,9 @@ public Object instanceMain(final String[] args, final List<String> packageList,
      * This method is not intended to be used outside of the GATK framework and tests.
      */
     public Object instanceMain(final String[] args) {
-        return instanceMain(args, getPackageList(), getClassList(), getCommandLineName());
+        final GATKConf configuration = getConfiguration();
+        FeatureManager.setConfiguration(configuration);
```

Comment:
I also move this part of the code to the other `instanceMain()` method, passing just the configuration object generated here instead of the list to set the `CommandLineProgram` configuration properly.